transport/tchannel/channel_outbound.go

```diff
@@ -37,6 +39,11 @@ var (
 	_ introspection.IntrospectableOutbound = (*ChannelOutbound)(nil)
 )
 
+const (
+	//TracingTagStatusCode is the span tag key for the YAPRC status code.
+	TracingTagStatusCode = "rpc.yarpc.status_code"
```

Comment:
How many copies of this tag do we have at this point? Let's extract it to some package and reuse.